### PR TITLE
Dev/bug fixes

### PR DESCRIPTION
**Pull request by armedturret**
- Fixed crashing when the user selects an id not displayed in movie search results
- Added transaction commits after account creation and login in case of a crash later in the program

### CODE REVIEW

**Comment by armedturret** [comment]
Don't approve this pr until this pr is approved: https://github.com/armedturret/sigma-db/pull/5

**Review by quappin — APPROVED** [review]
Looks good to me!

**Pull request merged**
